client/gutenberg/extensions/vr/editor.scss

```diff
@@ -0,0 +1,20 @@
+.wp-block-jetpack-vr {
+	position: relative;
+	max-width: 525px;
```

Comment:
I wonder where this value comes from.

Reply:
I think it's pretty arbitrary. Inclined to leave it for now.